### DISCUSSION TITLE
How to split long flextable in officedown powerpoint rmarkdown

### DISCUSSION THREAD

**Discussion by jic007** [davidgohel/officedown#114] [Q&A]
Suppose for some reason i need to print out the whole iris dataset in powerpoint rmakrdown

i need to do some text as an intdoduction at the beginning of the slide and need to then insert iris dataset. I should see the table split by say 12 rows a slide while repeating the title and adding (1 of 20)

been trying to find such a solution as i do some reports in powerpoint and some of them are 200 pages

Any ideas? Due to special requirement it must be done in powepoint

appreciate the help

**Comment by davidgohel** [comment]
No, see your question here : https://github.com/davidgohel/officedown/issues/115
